File: 01_ProgrammingBasics/_01_FirstStepsInProgrammingMoreExercies/_02_TriangleArea.java
package _01_ProgrammingBasics._01_FirstStepsInProgrammingMoreExercies;

import java.util.Scanner;

public class _02_TriangleArea {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input data
        double triangleSide = Double.parseDouble(scan.nextLine());
        double triangleHeight = Double.parseDouble(scan.nextLine());

        // Calculate the area of the triangle
        double triangleArea = triangleSide * triangleHeight / 2;

        // Print the calculated area with two decimal places
        System.out.printf("%.2f%n", triangleArea);

        scan.close();
    }
}

File: 23_C#Basics/_01_FirstStepsInProgrammingMoreExercises/_02_TriangleArea/_02_TriangleArea.cs
using System;

public class _02_TriangleArea
{
    public static void Main(string[] args)
    {
        // Read input data
        double triangleSide = double.Parse(Console.ReadLine());
        double triangleHeight = double.Parse(Console.ReadLine());

        // Calculate the area of the triangle
        double triangleArea = triangleSide * triangleHeight / 2;

        // Print the calculated area with two decimal places
        Console.WriteLine("{0:F2}\n", triangleArea);
    }
}